bno085:
  ros__parameters:
    ros_topic_prefix: "bno085/"
    connection_type: "i2c"
    data_query_frequency: 100
    calib_status_frequency: 1
    frame_id: "bno085"
    initial_calibration: False
    calibration_timeout: 60
    ## Sensor standard deviation [x,y,z]
    ## Used to calculate covariance matrices
    ## defaults are used if parameters below are not provided
    # variance_acc: [0.0, 0.0, 0.0] # [m/s^2]
    # variance_angular_vel: [0.0, 0.0, 0.0] # [rad/s]
    # variance_orientation: [0.0, 0.0, 0.0] # [rad]
    # variance_mag: [0.0, 0.0, 0.0] # [Tesla]
